<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>restore</class>
 <widget class="QWidget" name="restore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据恢复</string>
  </property>
  <widget class="QLineEdit" name="rst_res">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="res_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>68</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bu_res">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rst_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;恢复路径&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rst">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>进行恢复</string>
   </property>
  </widget>
  <widget class="QLabel" name="res_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;数据路径&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rst_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>118</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;数据恢复功能&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_main">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>323</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;解密密钥&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pwd_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>168</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <zorder>label_pic</zorder>
  <zorder>rst_res</zorder>
  <zorder>res_button</zorder>
  <zorder>bu_res</zorder>
  <zorder>rst_label</zorder>
  <zorder>rst</zorder>
  <zorder>res_label</zorder>
  <zorder>rst_button</zorder>
  <zorder>title</zorder>
  <zorder>return_main</zorder>
  <zorder>pwd_label</zorder>
  <zorder>password</zorder>
  <zorder>pwd_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
